packages/yourloops/pages/profile/credentials-form.tsx

```diff
@@ -69,12 +70,16 @@ function AuthenticationForm(props: AuthenticationFormProps): JSX.Element {
 
   return (
     <React.Fragment>
+      <div className={classes.categoryLabel}>
```

Comment:
I'm not sure to understand this. This is a UI modification right?

Reply:
moving the title into the components like the others